packages/grid/_modules_/grid/models/params/gridVirtualPageChangeParams.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Object passed as parameter of the virtual page change event.
+ */
+export interface GridVirtualPageChangeParams {
+  /**
+   * The index of the current page.
+   */
+  currentPage: number;
+  /**
+   * The index of the next page.
+   */
+  nextPage: number;
+  /**
+   * The index of the first row in the current page.
+   */
+  firstRowIndex: number;
+  /**
+   * The size of the virtual page.
+   */
+  pageSize: number;
```

Comment:
This seems more adequate

```suggestion
  /**
   * The index of the last row in the viewport.
   */
  lastRowIndex: number;
```